mbw.c

```diff
@@ -267,6 +319,7 @@ int main(int argc, char **argv)
     if(!quiet) {
         printf("Long uses %d bytes. ", long_size);
         printf("Allocating 2*%lld elements = %lld bytes of memory.\n", asize, 2*asize*long_size);
```

Comment:
todo: Adjust this for the number of threads (if taking the per-thread-asize allocation suggestion above).